bark/benchmark/scenario_dumper.py

```diff
@@ -110,3 +111,56 @@ def write_map(self, config_idx, folder):
       mapfile = params["Scenario"]["Generation"]["ConfigurableScenarioGeneration"]["MapFilename"]
       src = os.path.join("src", "database", mapfile)
       copyfile(src, os.path.join(folder, ntpath.basename(mapfile)))
+
+
+class OpenScenarioDumper(ScenarioDumper):
+  def __init__(self, base_result_folder, benchmark_result):
+    super(OpenScenarioDumper, self).__init__(
+      base_result_folder, benchmark_result)
+  
+  def GetTrajectoryPerAgent(self, world):
+    trajectory_per_agents = {}
+    for (agent_id, agent) in world.agents.items():
+      trajectory_per_agents[agent_id] = []
+      for state_action in agent.history:
```

Comment:
if history is too small, we are not saving everything. Do we have a mechanism to check for that?

Reply:
mhm, but where is defined what the expected history is? I also copy and pasted the cod here.

Reply:
I have implemented a log message for now; but it also would be possible to throw an exception.